src/lib/routes/admin-api/context.ts

```diff
@@ -189,13 +194,19 @@ export class ContextController extends Controller {
 
     async createContextField(
         req: IAuthRequest<void, void, UpsertContextFieldSchema>,
-        res: Response,
+        res: Response<ContextFieldSchema>,
     ): Promise<void> {
         const value = req.body;
         const userName = extractUsername(req);
 
-        await this.contextService.createContextField(value, userName);
-        res.status(201).end();
+        const result = await this.contextService.createContextField(
+            value,
+            userName,
+        );
+        res.status(201)
+            .header('location', `context/${result.name}`) // todo: how to ensure that the location is (and stays) correct?
```

Comment:
This is a discussion point that we should think about. Do we store api paths as variables anywhere? It would be nice to have this auto-update in case things change later.

Reply:
Yeah, this would be really nice! I do think this is a pretty unlikely situation (always nice to be confident though), changing APIs is rare because of compatibility reasons. Don't think we have a set of paths in variables right now, they're mostly set in strings where they're needed. No reason we can't make that happen, just needs time

Reply:
Yeah, fair. In that case, I think I'll leave it for now. With how unlikely it is to change, I think it makes sense to leave it as is.